perfkitbenchmarker/benchmark_spec.py

```diff
@@ -30,10 +30,13 @@
 import perfkitbenchmarker.deployment.shared.ini_constants as ini_constants
 from perfkitbenchmarker.gcp import gce_network
 from perfkitbenchmarker.gcp import gce_virtual_machine
+from perfkitbenchmarker.openstack import os_network
+from perfkitbenchmarker.openstack import os_virtual_machine
 
 GCP = 'GCP'
 AZURE = 'Azure'
 AWS = 'AWS'
+OS = 'OpenStack'
```

Comment:
Would you be willing to replace "OS" with "OPENSTACK" throughout this code? "OS" is a very confusing abbreviation, especially when used in a file like this that's shared among providers.